media-sound/fmit/fmit-1.0.15-r1.ebuild

```diff
@@ -0,0 +1,58 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+inherit qmake-utils
+
+MY_PN="v${PV}"
+
+DESCRIPTION="Free Music Instrument Tuner"
+HOMEPAGE="https://gillesdegottex.github.io/fmit"
+SRC_URI="https://github.com/gillesdegottex/fmit/archive/${MY_PN}.tar.gz \
+							-> ${P}.tar.gz"
+
+LICENSE="GPL-2+ LGPL-2.1"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="alsa jack oss portaudio"
+
+RDEPEND=">=sci-libs/fftw-3.3.4:3.0
```

Comment:
When adding the `SLOT`, you forgot the slot operator `=` :smile:, you had it in the previous version, go for >=sci-`libs/fftw-3.3.4:3.0=`